### DISCUSSION TITLE
ScriptReference/PlayerSettings.Android

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#389] [Notes]
# ScriptReference/PlayerSettings.Android

https://docs.unity3d.com/ScriptReference/PlayerSettings.Android.html

**Comment by magnoahlia** [comment]
If you're trying to get `BuildPipeline.BuildPlayer` to emit a Gradle project instead of an apk, you need to add `BuildOptions.AcceptExternalModificationsToPlayer` to the `BuildPlayerOptions`.   
See this thread: [Automate export of 'Google Android Project'](https://forum.unity.com/threads/unity-4-2-automate-export-of-google-android-project.194963/)
